packages/blockchain-wallet-v4-frontend/src/scenes/Login/Wallet/CheckEmail/index.tsx

```diff
@@ -62,11 +85,37 @@ const CheckEmail = (props: Props) => {
           fullwidth
           height='48px'
           data-e2e='loginResendEmail'
+          disabled={disabled && !hasErrorAlert}
+          // @ts-ignore
+          onClick={(e: SyntheticEvent) => {
+            setDisabled(true)
+            setSentState('sent')
```

Comment:
Weren't we getting a weird error until we flipped setDisabled and setSentState lines? Is it working now?